src/uniprotkb/config/UniProtKBColumnConfiguration.tsx

```diff
@@ -204,82 +201,68 @@ UniProtKBColumnConfiguration.set(UniProtKBColumn.length, {
 });
 
 UniProtKBColumnConfiguration.set(UniProtKBColumn.genePrimary, {
-  label: 'Gene names (Primary)',
+  label: 'Gene Names (Primary)',
   render(data) {
     const { geneNamesData } = data[EntrySection.NamesAndTaxonomy];
-    return (
-      <ExpandableList descriptionString="names" displayNumberOfHiddenItems>
-        {geneNamesData &&
-          geneNamesData.map(
-            (geneData) =>
-              geneData.geneName && (
-                <div key={geneData.geneName.value}>
-                  {geneData.geneName.value}
-                </div>
-              )
-          )}
+
+    const names = geneNamesData?.map((geneNames) => geneNames.geneName?.value);
+
+    return names?.length ? (
```

Comment:
I see the pattern `predicate ? true-branch : null` for all of the column modifications in this PR - is there a reason this is used rather `predicate && true-branch` (which I personally find a little bit easier to read)?

Reply:
This is because these are lengths, so you could end up displaying "0".
I could also test on the array itself but not its length, but I think the Expandable list renders an empty list, but I want it to render nothing, not an empty `<ul>`.

Reply:
thoughts on?
```
    return (
      Boolean(names?.length) && (
        <ExpandableList descriptionString="gene names" numberCollapsedItems={1}>
          {names}
        </ExpandableList>
      )
    );
```

Reply:
I would prefer:
```
return (
      names && (
        <ExpandableList descriptionString="gene names" numberCollapsedItems={1}>
          {names}
        </ExpandableList>
      )
    );
```
And modify ExpandableList in Franklin to return null when the number of children is 0. What do you think?

Reply:
That sounds good - also names could potentially include `undefined` elements so could determine if this is the case within franklin too